services/destination-connection-tester/destination_connection_tester.go

```diff
@@ -157,9 +162,8 @@ func TestBatchDestinationConnection(destination backendconfig.DestinationT) stri
 	testFileName := createTestFileForBatchDestination(destination.ID)
 	keyPrefixes := []string{config.GetEnv("RUDDER_CONNECTION_TESTING_BUCKET_FOLDER_NAME", misc.RudderTestPayload), destination.ID, time.Now().Format("01-02-2006")}
 	_, err := uploadTestFileForBatchDestination(testFileName, keyPrefixes, destination.DestinationDefinition.Name, destination)
-	var error string
 	if err != nil {
-		error = err.Error()
+		return err.Error()
 	}
-	return error
+	return ""
```

Comment:
why not `nil`?

Reply:
why not `""`? :stuck_out_tongue:

Reply:
oh, I'm sorry I was carried away by the name of the previous variable (`error`)...